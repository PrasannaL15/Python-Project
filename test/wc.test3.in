This text contains a single, exceptionally long sentence that extends over multiple lines and includes various punctuation marks, making it a challenge for the word count utility to accurately count the words in a non-standard sentence structure.