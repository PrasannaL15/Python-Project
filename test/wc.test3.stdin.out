       1      37     246

